### ISSUE TITLE
Hyperparameter ballpark for non-image envs

### ISSUE THREAD

**Issue by jendelel** [google-research/planet#49]
Hi,

I would like to use PlaNet for planning in maze-like environments. However, I can't get it to work reasonably even on classic gym Mujoco tasks such as Ant-v2 (non image version). Could you give me some intuition for the hyperparameters when running on non image environments? There are many hyperparameters and I don't know where to start tuning.

This is what I tried:
```
         id: AntLargeBeliefNew/
                          env: Ant-v2
                          symbolic_env: True
                          max_episode_length: 1000
                          experience_size: 1000000
                          activation_function: relu
                          embedding_size: 64
                          hidden_size: 64
                          belief_size: 30
                          state_size: 30
                          action_repeat: 1 (already part of openai gym)
                          action_noise: 0.3
                          episodes: 1000
                          collect_interval: 50
                          batch_size: 50
                          chunk_size: 50
                          overshooting_distance: 50
                          overshooting_kl_beta: 0
                          overshooting_reward_scale: 0
                          global_kl_beta: 0
                          free_nats: 3
                          bit_depth: 5
                          learning_rate: 0.001
                          learning_rate_schedule: 0
                          adam_epsilon: 0.0001
                          grad_clip_norm: 1000
                          planning_horizon: 12
                          optimisation_iters: 10
                          candidates: 1000
                          top_candidates: 100
                          test_interval: 15
                          test_episodes: 10
                          checkpoint_interval: 50
```

Thanks a lot for any help, 
Lukas

**Comment by danijar** [comment]
Does `collect_interval: 50` means to collect one episode after every gradient step? That would be too frequent, although we found that collecting data more frequently than when training from vision helps on low-dim tasks.

**Comment by jendelel** [comment]
Hi, 
thanks for the reply. 
collect interval corresponds to the Collect interval C from the algorithm from the paper (screenshot below)
![image](https://user-images.githubusercontent.com/16251857/64253233-213a9780-cf1d-11e9-8fd5-03a386a23b5a.png)
I think I tried even C=100, but the results were not very good. 

Lukas

**Comment by danijar** [comment]
I see. Just to make sure, note that `collect_every` in my code is `C * B` in the paper.

I'd try setting the model size back to the defaults and collecting data a bit more frequently.

**Comment by jendelel** [comment]
Thanks, I will try.
